AAI/AAIConnectProfile.md

```diff
@@ -30,50 +29,62 @@ systems.
+be interpreted as described in [[RFC2119]](#ref-rfc2119).
 
 {% hr2 %}
 
 ## Terminology
 
-<a name="term-jwt"></a>
-**JWT** -- JSON Web Token as defined in [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519).
-A JWT contains a set of [claims](https://datatracker.ietf.org/doc/html/rfc7519#section-2).
-A claim is a piece of information asserted about a subject, represented as a name/value pair consisting of
-a claim name (a string) and a claim value (any JSON value). This definiton of claims
-is inherited by [OIDC](http://openid.net/specs/openid-connect-core-1_0.html) from RFC7519.
+This specification inherits terminology from the OpenID
+Connect [[OIDC-Core]](#ref-oidc-core) 
+and OAuth 2.0 Framework [[RFC6749]](#ref-rfc6749) specifications.
+
+<a name="term-broker"></a> **Broker** -- An OIDC Provider service that
+authenticates a user (potentially by relying on an [Identity Provider](#term-identity-provider)), 
+collects user's [Visas](#term-visa) from internal and/or external [Visa Issuers](#term-visa-issuer), 
+and provides them to [Passport Clearinghouses](#term-passport-clearinghouse).
+Brokers may also be Passport Clearinghouses of other upstream Brokers.
 
-<a name="term-ga4gh-claim"></a> **GA4GH Claim** -- A JWT claim as defined by a GA4GH
+**Claim**{: #term-claim} -- as [defined](https://datatracker.ietf.org/doc/html/rfc7519#section-2) 
+by the JWT specification [[RFC7519]](#ref-rfc7519) -- A piece of information asserted about a subject, 
+represented as a name/value pair consisting of
+a claim name (a string) and a claim value (any JSON value). 
+
+<a name="term-data-holder"></a> **Data Holder** -- An organization that
+holds a specific set of data (or its copy) and respects
+and enforces the Data Access Committee's (DAC's) decisions on who can access it. A DAC
+can also be a [Data Holder](#term-data-holder). A [Data Holder](#term-data-holder) runs a
+[Passport Clearinghouse Server](#term-passport-clearinghouse) at a minimum.
+
+<a name="term-ga4gh-claim"></a>
+**GA4GH Claim** -- A [Claim](#term-claim) as defined by a GA4GH
 documented technical standard that is making use of this AAI specification. Typically
-this is the `ga4gh_passport_v1` or `ga4gh_visa_v1` claim for GA4GH Passports v1.x (see 
-[GA4GH DURI Passport](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md)).
-Note that GA4GH is not the organization making the claim nor taking responsibility
-for the claim as this is a reference to a GA4GH documented standard only.
+this is the `ga4gh_passport_v1` or `ga4gh_visa_v1` [Claim](#term-claim) for [Passports v1.x](#term-passport).
+A GA4GH Claim is asserted by the entity that signed the token in which it is contained (not by GA4GH).
 
-<a name="term-identity-provider"></a> **Identity Provider (IdP)** -- a
+<a name="term-identity-provider"></a> **Identity Provider (IdP)** -- defined as OpenID Provider 
```

Comment:
I think that IdP should not be defined narrowly as an OpenID Provider, because in ELIXIR the big majority of IdPs are SAML-based IdPs, and SAML-to-OIDC bridges are used to translate the protocols.  I would leave the definition as it was, defining only the function of providing a user's identity, without specifying the protocol used.